packages/debug/src/index.ts

```diff
@@ -0,0 +1,255 @@
+/* eslint-disable no-console */
+import { Signal, Effect, Computed } from "@preact/signals-core";
+import { formatValue, getSignalName } from "./utils";
+import { UpdateInfo, Node, Computed as ComputedType } from "./internal";
+
+const inflightUpdates = new Set<Signal>();
+const updateInfoMap = new WeakMap<Signal, UpdateInfo[]>();
+const trackers = new WeakMap<Signal, number>();
+const activeSignals = new Set<Signal>();
+const signalValues = new WeakMap<Signal, any>();
+const subscriptions = new WeakMap<Signal, () => void>();
+
+export function getDebugStats() {
```

Comment:
Probably worth removing this now, mainly used this to see whether we leaked memory